web/cobrands/sass/_base.scss

```diff
@@ -34,6 +34,12 @@ $geolocation-link-border: $geolocation-link !default;
 @import "_mixins";
 @import "_report_list";
 
+// Prevent glow/bounce at bottom of page (also sadly prevents pull to refresh
+// at top, but refresh button still there)
+html {
+  overscroll-behavior-y: none;
```

Comment:
Hmm, I actually prefer the rubber banding (and funnily enough the [PWA Builder code does disable it](https://github.com/pwa-builder/pwabuilder-ios/blob/1f8ff59ee554137b34bb616869f5e4e62dc2754f/Microsoft.PWABuilder.IOS.Web/Resources/ios-project-src/pwa-shell/WebView.swift#L31) by default and I re-enabled it when I built the app). Most scrollviews on iOS apps do the rubberbanding by default as far as I can tell.

Reply:
I think I noticed this when making a new report, and e.g. scrolling down the category list and being annoyed by it bouncing at the end, so took longer to 'settle' and then press the continue button - I don't find apps generally bounce for that kind of thing, but I admit I don't use a lot of apps :) And scrolling the front page, seemed odd to continue past the footer. Dunno.